content/en/content-management/related.md

```diff
@@ -14,13 +14,18 @@ aliases: [/content/related/,/related/]
 toc: true
 ---
 
+
 {{% note %}}
 We currently do not index **Page content**. We thought we would release something that will make most people happy before we start solving [Sherlock's last case](https://github.com/joearms/sherlock).
 {{% /note %}}
 
 ## List Related Content
 
-To list up to 5 related pages is as simple as including something similar to this partial in your single page template:
+Hugo uses "indices" based on front matter parameters to identify related content. By default, it creates indices for the "**keywords**" and "**date**" parameters but you can create an index for any parameters you like (see below). Read the ["related" package documentation](https://godoc.org/github.com/gohugoio/hugo/related#pkg-variables) for more. The default behavior is to **exclude** related posts which are _newer_ than the current post. This can be configured via _includeNewer_ in your site-wide `config.toml` (see below).
```

Comment:
I was tempted to link to tier doc before but I don't think it's a good idea. The preceding sentence is plenty enough.
As a user, I would not mind reading the default configuration as a `yaml` example though. This way the reader gets to see the default configuration and a configuration example. One 🐦, Two 🗿  ... 😄